simperby/src/node.rs

```diff
@@ -1,12 +1,14 @@
+mod history_storage;
 mod network;
-mod storage;
+mod state_storage;
+mod stf;
```

Comment:
what does stf stands for?

Reply:
@byeongjee I think it stands for state transition functions.

Reply:
state_transition_function is too long I guess. Let's use `stf`.

Reply:
I think state_transition_function is better. Not many will get stf before opening stf.rs file.

Reply:
I'm with @TomTaehoonKim on this

Reply:
updated to `state_transition.rs`

Reply:
@junha1 `stf.rs` to `state_transition.rs`? Is it pushed? I can't see it in files changed now.

Reply:
fixed